packages/core/admin/admin/src/content-manager/components/BlocksEditor/plugins/withLinks.js

```diff
@@ -0,0 +1,35 @@
+import { Path } from 'slate';
+
+const withLinks = (editor) => {
+  const { isInline, apply } = editor;
+
+  // Links are inline elements, so we need to override the isInline method for slate
+  editor.isInline = (element) => {
+    return element.type === 'link' ? true : isInline(element);
+  };
+
+  // We keep a track of the last inserted link path
+  // So we can show the popover on the link component if that link is the last one inserted
+  editor.lastInsertedLinkPath = null;
```

Comment:
Is this a property we are creating or overriding?

Reply:
We are creating it

Reply:
I'm wondering how we could document that because maybe it will get confusing later on. Someone in the future might look for this in the Slate docs and not understand why it isn't there for example.